src/coreclr/jit/compiler.cpp

```diff
@@ -5935,6 +5941,12 @@ void Compiler::generatePatchpointInfo()
     JITDUMP("--OSR-- Tier0 callee saves: ");
     JITDUMPEXEC(dspRegMask((regMaskTP)patchpointInfo->CalleeSaveRegisters()));
     JITDUMP("\n");
+#elif defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
+    // NOTE:
+    //    The callee saved registers include the FP/RA here!!!
+    //    We save the first callee saved register's offset, which is the FP's offset.
+    patchpointInfo->SetCalleeSaveRegisters(codeGen->genSPtoFPdelta());
```

Comment:
`CalleeSaveRegisters` is supposed to be a register mask. This PR starts using it for something else. Why is it necessary to save this offset in this way?
This kind of thing makes long-term maintenance hard for us.

Reply:
> `CalleeSaveRegisters` is supposed to be a register mask. This PR starts using it for something else. Why is it necessary to save this offset in this way? This kind of thing makes long-term maintenance hard for us.

As I don't want to add new interface.
If needed, I can add a new one.